lib/sandbox.js

```diff
@@ -1496,11 +1497,12 @@ module.exports.cli = {
          *
          * @param {String} spec the sandbox id or tenant
          * @param {String} alias the alias name to register for the sandbox
+         * @param {Boolean} unique optional flag to configure the alias to be unique, false by default
          * @param {Boolean} asJson optional flag to force output in json, false by default
          */
-        create : function (spec, alias, asJson) {
+        create : function (spec, alias, unique, asJson) {
             runForSandbox(spec, asJson, function (sandbox) {
-                registerForSandbox(sandbox, alias, function (err, result) {
+                registerForSandbox(sandbox, alias, unique, function (err, result) {
```

Comment:
The "unique" handling is a bit different from the the non-unique default case. Here we don't have a registration link anymore, but we get a verification record back
![Screenshot 2021-11-01 at 07 53 29](https://user-images.githubusercontent.com/42992747/139633431-3bf6ebab-8e59-4861-94b7-e2f679eefc9f.png):

Reply:
@martin-dodo-hartmann Good catch, will look into this and update the PR